docs/epics.md

```
@@ -774,8 +774,8 @@ So that I understand how to get started.
 **Given** I am a new user with zero VM requests
 **When** I view the dashboard
 **Then** I see an empty state illustration with:
-  - Friendly message: "Noch keine VMs angefordert"
-  - Clear CTA: "Erste VM anfordern" button
+  - Friendly message: "No VMs requested yet"
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)